xonsh/webconfig/routes.py

```diff
@@ -239,10 +240,13 @@ def get(self):
 
     def post(self, data: "cgi.FieldStorage"):
         if data:
-            prompt = data.getvalue(self.var_name)
+            prompt = data.getvalue(self.var_name).replace("\r", "")
             self.env[self.var_name] = prompt
             self.update_rc(prompt=prompt)
 
+    def ee(self, str):
+        return re.sub(r"\\r", "", str)
+
```

Comment:
Is this being used or is this left over from debugging?

Reply:
removed